### PR TITLE
Malorian Arms 3516 and Johnny Robohand bundle

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## Changelog
:cl:
add: Malorian Arms 3516(Johnny Robohands pistol)
add: Johnny Silverhand traitor kit for 45TC
/:cl:

## Malorian Arms 3516
image

Chambered in explosive 14mm and custom made, this thing is the dogs' bollocks of pistols. It is by far the best gun you can get, however it does have a few caveats. Namely; you require the arm you are holding it with to be robotic, if you don't have a robotic arm, you cannot use the gun.

Basic statistics:
Mag capacity: 10 rounds
Calibre: 14mm
Damage: 60
These rounds are huge and have a high delimb chance.

Little preview of the gun(damage done is out of date):
https://streamable.com/9yncms

Malorian Arms 3516 credits:
Code: Me
Sprites: Investigator77
Sound: Cyberpunk 2077(with edits)

## The Robohand bundle
This PR also adds the Johnny Robohand bundle which contains multiple useful assets that will help you be more like the real thing. It's 45 TC and comes with the kit to back that up.

It contains:
1x Malorian Arms 3516
4x 14mm magazines
1x Chest rig
1x Robohand pants
1x Combat gloves
1x Combat boots
1x Robohand sunglasses
1x Robohand Bulletproof armor
1x Robotic arm replacer
1x Energysword implanter
1x Nodrop implant for yer' big old force multiplier

This also adds bodypart autosurgeons... Not sure what else to use em for, but they exist now.

### CODE REVIEW

**Comment by Draganfrukts** [comment]
If I need your body I'll fuck it.

**Comment by Swiftfeather** [comment]
Think we could add stam damage protection to the vest so they don't get "hehe funni stamcrit"-ed?

**Comment by Gandalf2k15** [comment]
For those that think this is OP:
Ion's insta kill you
You have no anti-stun
You have mediocre armour
You need a robotic arm to use the gun
It costs all of your TC

**Comment by Gandalf2k15** [comment]
Wound bonus is removed.

**Comment by Gandalf2k15** [comment]
They aren't explosive now.

**Comment by gamermomentum** [comment]
This looks pretty wacky. 
Might wanna give a big think to whether this can go on sale or not because if you could combo this with some other strong shit it'd be incredibly wacky.

**Comment by Gandalf2k15** [comment]
> This looks pretty wacky.
> Might wanna give a big think to whether this can go on sale or not because if you could combo this with some other strong shit it'd be incredibly wacky.

Impossible. It costs your 45tc. Nothing else can be bought.

File: modular_skyrat/modules/3516/code/silverhand_bundle.dm [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	new /obj/item/ammo_box/magazine/m14mm(src)
+	new /obj/item/ammo_box/magazine/m14mm(src)
+	new /obj/item/storage/belt/military(src)
+	new /obj/item/clothing/under/pants/black/robohand(src)
+	new /obj/item/clothing/gloves/combat(src)
+	new /obj/item/clothing/shoes/combat(src)
+	new /obj/item/clothing/glasses/sunglasses/robohand(src)
+	new /obj/item/clothing/suit/armor/bulletproof/robohand(src)
+	new /obj/item/autosurgeon/bodypart/r_arm_robotic(src)
+	new /obj/item/autosurgeon/organ/syndicate/esword_arm(src)
```
Comment (Ziiro) [inline modular_skyrat/modules/3516/code/silverhand_bundle.dm:14]: Not sure the esword arm is necessary or flavorful. Standard CQC might be more fitting to the kit.

**Comment by Ziiro** [comment]
Mechanically and thematically I fucking love this, but I'm not sure the kit works for skyrat since this thing is 200% murderbone.

Also super limited ammo? Might want to add the new bullets to the hacked autolathe or something.

**Comment by KasparoVy** [comment]
Man this is kickass, fuuuuck yeah. I love the wee spin it does when loading, ahaha.

**Comment by PCPort** [comment]
> Mechanically and thematically I fucking love this, but I'm not sure the kit works for skyrat since this thing is 200% murderbone.
> 
> Also super limited ammo? Might want to add the new bullets to the hacked autolathe or something.

I *already* have an idea for how to work this into an antag round while keeping things entertaining. Might require some admin co-op, though.

**Pull request merged**